osu.Game/Tests/Visual/SkinnableTestScene.cs

```diff
@@ -120,6 +118,14 @@ private Drawable createProvider(Skin skin, Func<Drawable> creationFunction, IBea
             };
         }
 
+        /// <summary>
+        /// Creates the ruleset for adding the corresponding skin transforming component.
+        /// </summary>
+        [NotNull]
+        protected abstract Ruleset CreateRulesetForSkinProvider();
```

Comment:
doesn't match other methods. should be `CreateSinProviderRuleset` or similar.

Reply:
was following up with its neighbour method (`CreateBeatmapForSkinProvider`)

Reply:
fair enough